scripts/devnet/.env

```diff
@@ -2,6 +2,7 @@ LOTUS_IMAGE=ghcr.io/chainsafe/lotus-devnet:2024-07-03-1f030c2
 FOREST_DATA_DIR=/forest_data
 LOTUS_DATA_DIR=/lotus_data
 FIL_PROOFS_PARAMETER_CACHE=/var/tmp/filecoin-proof-parameters
+LOTUS_SYNC_BOOTSTRAP_PEERS=1
```

Comment:
there's no need to do it here, 2k networks have this value at `1` by default.

Reply:
Yes but it doesn't hurt right?

Reply:
It looks like we are overriding a default behaviour (but we don't), it doesn't bring any value in this context, while only increasing the complexity if anyone is analysing it. That said, it's just a personal preference, not a huge deal.